### ISSUE TITLE
Flutter build hangs on: Observatory listening on 

### ISSUE THREAD

**Issue by nani-samireddy** [flutter/flutter#81196]
I have been facing this issue of Observatory listening on ************************************ 
below i paste some info....

<details>
<summary>flutter doctor -v</summary>

```bash
C:\Users\NANI>flutter doctor -v
[√] Flutter (Channel stable, 2.0.5, on Microsoft Windows [Version 10.0.19042.928], locale en-IN)
    • Flutter version 2.0.5 at C:\flutter
    • Framework revision adc687823a (10 days ago), 2021-04-16 09:40:20 -0700
    • Engine revision b09f014e96
    • Dart version 2.12.3

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\NANI\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.55.2)
    • VS Code at C:\Users\NANI\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.21.0

[√] Connected device (3 available)
    • vivo 1917 (mobile) • d053be10 • android-arm64  • Android 11 (API 30)
    • Chrome (web)       • chrome   • web-javascript • Google Chrome 90.0.4430.85
    • Edge (web)         • edge     • web-javascript • Microsoft Edge 89.0.774.68

• No issues found!
```
</details>

<details>
<summary>results of flutter run -v</summary>

```bash
C:\Users\NANI\AndroidStudioProjects\christain_lyrics>flutter pub get
Running "flutter pub get" in christain_lyrics...                   10.8s
C:\Users\NANI\AndroidStudioProjects\christain_lyrics>flutter run -v
[ +103 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +123 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] adc687823a831bbebe28bdccfac1a628ca621513
[        ] executing: [C:\flutter/] git tag --points-at adc687823a831bbebe28bdccfac1a628ca621513
[  +84 ms] Exit code 0 from: git tag --points-at adc687823a831bbebe28bdccfac1a628ca621513
[        ] 2.0.5
[  +52 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +77 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [C:\flutter/] git ls-remote --get-url origin
[  +69 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +127 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[  +79 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +123 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +7 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +89 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +70 ms] List of devices attached
           d053be10               device product:1917 model:vivo_1917 device:1917 transport_id:1
[   +8 ms] C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell getprop
[ +126 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +7 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +131 ms] Skipping pub get: version match.
[  +54 ms] Found plugin path_provider_linux at C:\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-0.0.1+2\
[  +11 ms] Found plugin path_provider_windows at C:\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-0.0.5\
[  +19 ms] Found plugin shared_preferences at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-0.5.12+4\
[   +5 ms] Found plugin shared_preferences_linux at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-0.0.2+4\
[   +4 ms] Found plugin shared_preferences_macos at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_macos-0.0.1+11\
[   +7 ms] Found plugin shared_preferences_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-0.1.2+7\
[   +4 ms] Found plugin shared_preferences_windows at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-0.0.2+3\
[ +156 ms] Found plugin path_provider_linux at C:\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_linux-0.0.1+2\
[   +4 ms] Found plugin path_provider_windows at C:\flutter\.pub-cache\hosted\pub.dartlang.org\path_provider_windows-0.0.5\
[   +8 ms] Found plugin shared_preferences at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences-0.5.12+4\
[   +2 ms] Found plugin shared_preferences_linux at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_linux-0.0.2+4\
[   +2 ms] Found plugin shared_preferences_macos at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_macos-0.0.1+11\
[   +3 ms] Found plugin shared_preferences_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_web-0.1.2+7\
[   +2 ms] Found plugin shared_preferences_windows at C:\flutter\.pub-cache\hosted\pub.dartlang.org\shared_preferences_windows-0.0.2+3\
[  +35 ms] Generating C:\Users\NANI\AndroidStudioProjects\christain_lyrics\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[  +89 ms] ro.hardware = qcom
[  +70 ms] Initializing file store
[  +13 ms] Skipping target: gen_localizations
[   +6 ms] complete
[   +5 ms] Launching lib\main.dart on vivo 1917 in debug mode...
[   +9 ms] C:\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata --output-dill
C:\Users\NANI\AppData\Local\Temp\flutter_tools.42f10754\flutter_tool.3762756b\app.dill --packages C:\Users\NANI\AndroidStudioProjects\christain_lyrics\.dart_tool\package_config.json
-Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build\cache.dill.track.dill
[  +23 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\build-tools\30.0.3\aapt dump xmltree
C:\Users\NANI\AndroidStudioProjects\christain_lyrics\build\app\outputs\flutter-apk\app.apk AndroidManifest.xml
[  +70 ms] Exit code 0 from: C:\Users\NANI\AppData\Local\Android\sdk\build-tools\30.0.3\aapt dump xmltree
C:\Users\NANI\AndroidStudioProjects\christain_lyrics\build\app\outputs\flutter-apk\app.apk AndroidManifest.xml
[   +1 ms] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1e
               A: android:compileSdkVersionCodename(0x01010573)="11" (Raw: "11")
               A: package="com.nanisamireddy.christain_lyrics" (Raw: "com.nanisamireddy.christain_lyrics")
               A: platformBuildVersionCode=(type 0x10)0x1e
               A: platformBuildVersionName=(type 0x10)0xb
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x1d
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1e
               E: uses-permission (line=14)
                 A: android:name(0x01010003)="android.permission.INTERNET" (Raw: "android.permission.INTERNET")
               E: application (line=16)
                 A: android:label(0x01010001)="christain_lyrics" (Raw: "christain_lyrics")
                 A: android:icon(0x01010002)=@0x7f080000
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 A: android:extractNativeLibs(0x010104ea)=(type 0x12)0x0
                 A: android:appComponentFactory(0x0101057a)="androidx.core.app.CoreComponentFactory" (Raw: "androidx.core.app.CoreComponentFactory")
                 E: activity (line=22)
                   A: android:theme(0x01010000)=@0x7f0a0000
                   A: android:name(0x01010003)="com.nanisamireddy.christain_lyrics.MainActivity" (Raw: "com.nanisamireddy.christain_lyrics.MainActivity")
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type 0x11)0x40003fb4
                   A: android:windowSoftInputMode(0x0101022b)=(type 0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type 0x12)0xffffffff
                   E: meta-data (line=36)
                     A: android:name(0x01010003)="io.flutter.embedding.android.NormalTheme" (Raw: "io.flutter.embedding.android.NormalTheme")
                     A: android:resource(0x01010025)=@0x7f0a0001
                   E: meta-data (line=46)
                     A: android:name(0x01010003)="io.flutter.embedding.android.SplashScreenDrawable" (Raw: "io.flutter.embedding.android.SplashScreenDrawable")
                     A: android:resource(0x01010025)=@0x7f040000
                   E: intent-filter (line=50)
                     E: action (line=51)
                       A: android:name(0x01010003)="android.intent.action.MAIN" (Raw: "android.intent.action.MAIN")
                     E: category (line=53)
                       A: android:name(0x01010003)="android.intent.category.LAUNCHER" (Raw: "android.intent.category.LAUNCHER")
                 E: meta-data (line=60)
                   A: android:name(0x01010003)="flutterEmbedding" (Raw: "flutterEmbedding")
                   A: android:value(0x01010024)=(type 0x10)0x2
[   +9 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell -x logcat -v time -t 1
[  +38 ms] <- compile package:christain_lyrics/main.dart
[ +202 ms] --------- beginning of main
                    04-26 17:01:27.310 V/MediaPlayerNative(19887): disconnect
[  +20 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe version
[  +59 ms] Android Debug Bridge version 1.0.41
           Version 31.0.2-7242960
           Installed as C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe
[   +5 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe start-server
[ +117 ms] Building APK
[  +27 ms] Running Gradle task 'assembleDebug'...
[  +24 ms] Using gradle from C:\Users\NANI\AndroidStudioProjects\christain_lyrics\android\gradlew.bat.
[   +3 ms] C:\Users\NANI\AndroidStudioProjects\christain_lyrics\android\gradlew.bat mode: 33279 rwxrwxrwx.
[  +21 ms] executing: C:\Program Files\Android\Android Studio\jre\bin\java -version
[ +245 ms] Exit code 0 from: C:\Program Files\Android\Android Studio\jre\bin\java -version
[   +1 ms] openjdk version "1.8.0_242-release"
           OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
           OpenJDK 64-Bit Server VM (build 25.242-b01, mixed mode)
[   +4 ms] executing: [C:\Users\NANI\AndroidStudioProjects\christain_lyrics\android/] C:\Users\NANI\AndroidStudioProjects\christain_lyrics\android\gradlew.bat -Pverbose=true
-Ptarget-platform=android-arm64 -Ptarget=C:\Users\NANI\AndroidStudioProjects\christain_lyrics\lib\main.dart -Ptrack-widget-creation=true -Pfilesystem-scheme=org-dartlang-root assembleDebug
[+12201 ms] > Task :app:compileFlutterBuildDebug
[  +13 ms] [ +315 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +338 ms] [ +504 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +14 ms] [        ] adc687823a831bbebe28bdccfac1a628ca621513
[   +1 ms] [   +1 ms] executing: [C:\flutter/] git tag --points-at adc687823a831bbebe28bdccfac1a628ca621513
[ +384 ms] [ +395 ms] Exit code 0 from: git tag --points-at adc687823a831bbebe28bdccfac1a628ca621513
[  +21 ms] [        ] 2.0.5
[  +80 ms] [ +103 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +197 ms] [ +205 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +5 ms] [        ] origin/stable
[   +3 ms] [        ] executing: [C:\flutter/] git ls-remote --get-url origin
[ +176 ms] [ +172 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] [        ] https://github.com/flutter/flutter.git
[ +204 ms] [ +247 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[ +201 ms] [ +130 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] [        ] stable
[ +102 ms] [ +175 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +2 ms] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [   +9 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[  +21 ms] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +164 ms] [ +155 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +2 ms] [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] [   +2 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[  +18 ms] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +156 ms] [ +166 ms] Initializing file store
[   +3 ms] [  +35 ms] Done initializing file store
[  +89 ms] [ +103 ms] Skipping target: gen_localizations
[+2411 ms] [+2360 ms] Skipping target: kernel_snapshot
[ +902 ms] [ +952 ms] Skipping target: debug_android_application
[   +1 ms] [   +1 ms] Persisting file store
[        ] [  +23 ms] Done persisting file store
[  +91 ms] [  +23 ms] build succeeded.
[   +1 ms] [  +41 ms] "flutter assemble" took 3,976ms.
[   +1 ms] [   +7 ms] ensureAnalyticsSent: 0ms
[   +1 ms] [   +2 ms] Running shutdown hooks
[        ] [        ] Shutdown hooks complete
[   +1 ms] [   +6 ms] exiting with code 0
[+1201 ms] > Task :app:packLibsflutterBuildDebug UP-TO-DATE
[   +2 ms] > Task :app:preBuild UP-TO-DATE
[   +3 ms] > Task :app:preDebugBuild UP-TO-DATE
[   +3 ms] > Task :shared_preferences:preBuild UP-TO-DATE
[   +2 ms] > Task :shared_preferences:preDebugBuild UP-TO-DATE
[   +1 ms] > Task :shared_preferences:compileDebugAidl NO-SOURCE
[   +1 ms] > Task :app:compileDebugAidl NO-SOURCE
[   +1 ms] > Task :shared_preferences:packageDebugRenderscript NO-SOURCE
[   +1 ms] > Task :app:compileDebugRenderscript NO-SOURCE
[   +1 ms] > Task :app:generateDebugBuildConfig UP-TO-DATE
[   +1 ms] > Task :shared_preferences:writeDebugAarMetadata UP-TO-DATE
[ +498 ms] > Task :app:checkDebugAarMetadata UP-TO-DATE
[  +85 ms] > Task :app:cleanMergeDebugAssets
[   +3 ms] > Task :app:mergeDebugShaders UP-TO-DATE
[   +1 ms] > Task :app:compileDebugShaders NO-SOURCE
[  +86 ms] > Task :app:generateDebugAssets UP-TO-DATE
[   +1 ms] > Task :shared_preferences:mergeDebugShaders UP-TO-DATE
[   +1 ms] > Task :shared_preferences:compileDebugShaders NO-SOURCE
[   +1 ms] > Task :shared_preferences:generateDebugAssets UP-TO-DATE
[        ] > Task :shared_preferences:packageDebugAssets UP-TO-DATE
[ +188 ms] > Task :app:mergeDebugAssets
[+2708 ms] > Task :app:copyFlutterAssetsDebug
[   +1 ms] > Task :app:generateDebugResValues UP-TO-DATE
[   +1 ms] > Task :app:generateDebugResources UP-TO-DATE
[        ] > Task :shared_preferences:compileDebugRenderscript NO-SOURCE
[        ] > Task :shared_preferences:generateDebugResValues UP-TO-DATE
[        ] > Task :shared_preferences:generateDebugResources UP-TO-DATE
[ +494 ms] > Task :shared_preferences:packageDebugResources UP-TO-DATE
[ +187 ms] > Task :app:mergeDebugResources UP-TO-DATE
[   +2 ms] > Task :app:createDebugCompatibleScreenManifests UP-TO-DATE
[        ] > Task :app:extractDeepLinksDebug UP-TO-DATE
[        ] > Task :shared_preferences:extractDeepLinksDebug UP-TO-DATE
[   +1 ms] > Task :shared_preferences:processDebugManifest UP-TO-DATE
[ +104 ms] > Task :app:processDebugMainManifest UP-TO-DATE
[   +1 ms] > Task :app:processDebugManifest UP-TO-DATE
[        ] > Task :app:processDebugManifestForPackage UP-TO-DATE
[        ] > Task :shared_preferences:compileDebugLibraryResources UP-TO-DATE
[        ] > Task :shared_preferences:parseDebugLocalResources UP-TO-DATE
[  +90 ms] > Task :shared_preferences:generateDebugRFile UP-TO-DATE
[ +201 ms] > Task :app:processDebugResources UP-TO-DATE
[   +1 ms] > Task :shared_preferences:generateDebugBuildConfig UP-TO-DATE
[   +1 ms] > Task :shared_preferences:javaPreCompileDebug UP-TO-DATE
[        ] > Task :shared_preferences:compileDebugJavaWithJavac UP-TO-DATE
[        ] > Task :shared_preferences:bundleLibCompileToJarDebug UP-TO-DATE
[ +202 ms] > Task :app:compileDebugKotlin UP-TO-DATE
[   +1 ms] > Task :app:javaPreCompileDebug UP-TO-DATE
[   +1 ms] > Task :app:compileDebugJavaWithJavac UP-TO-DATE
[   +4 ms] > Task :app:compileDebugSources UP-TO-DATE
[   +1 ms] > Task :app:mergeDebugNativeDebugMetadata NO-SOURCE
[   +2 ms] > Task :app:compressDebugAssets UP-TO-DATE
[   +2 ms] > Task :app:processDebugJavaRes NO-SOURCE
[   +6 ms] > Task :shared_preferences:processDebugJavaRes NO-SOURCE
[  +16 ms] > Task :shared_preferences:bundleLibResDebug NO-SOURCE
[   +3 ms] > Task :app:mergeDebugJavaResource UP-TO-DATE
[  +54 ms] > Task :app:checkDebugDuplicateClasses UP-TO-DATE
[   +2 ms] > Task :app:desugarDebugFileDependencies UP-TO-DATE
[ +102 ms] > Task :app:mergeExtDexDebug UP-TO-DATE
[   +1 ms] > Task :shared_preferences:bundleLibRuntimeToJarDebug UP-TO-DATE
[        ] > Task :app:dexBuilderDebug UP-TO-DATE
[        ] > Task :app:mergeLibDexDebug UP-TO-DATE
[        ] > Task :app:mergeProjectDexDebug UP-TO-DATE
[        ] > Task :app:mergeDebugJniLibFolders UP-TO-DATE
[        ] > Task :shared_preferences:mergeDebugJniLibFolders UP-TO-DATE
[        ] > Task :shared_preferences:mergeDebugNativeLibs NO-SOURCE
[        ] > Task :shared_preferences:stripDebugDebugSymbols NO-SOURCE
[        ] > Task :shared_preferences:copyDebugJniLibsProjectOnly UP-TO-DATE
[  +88 ms] > Task :app:mergeDebugNativeLibs UP-TO-DATE
[   +1 ms] > Task :app:stripDebugDebugSymbols UP-TO-DATE
[        ] > Task :app:validateSigningDebug UP-TO-DATE
[   +1 ms] > Task :app:packageDebug UP-TO-DATE
[+5101 ms] > Task :app:assembleDebug
[   +1 ms] > Task :shared_preferences:copyDebugJniLibsProjectAndLocalJars UP-TO-DATE
[ +198 ms] > Task :shared_preferences:extractDebugAnnotations UP-TO-DATE
[   +1 ms] > Task :shared_preferences:mergeDebugGeneratedProguardFiles UP-TO-DATE
[        ] > Task :shared_preferences:mergeDebugConsumerProguardFiles UP-TO-DATE
[   +1 ms] > Task :shared_preferences:prepareLintJarForPublish UP-TO-DATE
[   +1 ms] > Task :shared_preferences:mergeDebugJavaResource UP-TO-DATE
[        ] > Task :shared_preferences:syncDebugLibJars UP-TO-DATE
[        ] > Task :shared_preferences:bundleDebugAar UP-TO-DATE
[        ] > Task :shared_preferences:compileDebugSources UP-TO-DATE
[        ] > Task :shared_preferences:assembleDebug UP-TO-DATE
[  +84 ms] Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
[   +2 ms] Use '--warning-mode all' to show the individual deprecation warnings.
[   +1 ms] See https://docs.gradle.org/6.8/userguide/command_line_interface.html#sec:command_line_warnings
[        ] BUILD SUCCESSFUL in 29s
[        ] 58 actionable tasks: 5 executed, 53 up-to-date
[ +618 ms] Running Gradle task 'assembleDebug'... (completed in 30.5s)
[ +454 ms] calculateSha: LocalDirectory: 'C:\Users\NANI\AndroidStudioProjects\christain_lyrics\build\app\outputs\flutter-apk'/app.apk
[ +594 ms] calculateSha: reading file took 592us
[+1369 ms] calculateSha: computing sha took 1367us
[  +11 ms] √ Built build\app\outputs\flutter-apk\app-debug.apk.
[  +15 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\build-tools\30.0.3\aapt dump xmltree C:\Users\NANI\AndroidStudioProjects\christain_lyrics\build\app\outputs\flutter-apk\app.apk AndroidManifest.xml
[  +68 ms] Exit code 0 from: C:\Users\NANI\AppData\Local\Android\sdk\build-tools\30.0.3\aapt dump xmltree C:\Users\NANI\AndroidStudioProjects\christain_lyrics\build\app\outputs\flutter-apk\app.apk
AndroidManifest.xml
[   +1 ms] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1e
               A: android:compileSdkVersionCodename(0x01010573)="11" (Raw: "11")
               A: package="com.nanisamireddy.christain_lyrics" (Raw: "com.nanisamireddy.christain_lyrics")
               A: platformBuildVersionCode=(type 0x10)0x1e
               A: platformBuildVersionName=(type 0x10)0xb
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x1d
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1e
               E: uses-permission (line=14)
                 A: android:name(0x01010003)="android.permission.INTERNET" (Raw: "android.permission.INTERNET")
               E: application (line=16)
                 A: android:label(0x01010001)="christain_lyrics" (Raw: "christain_lyrics")
                 A: android:icon(0x01010002)=@0x7f080000
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 A: android:extractNativeLibs(0x010104ea)=(type 0x12)0x0
                 A: android:appComponentFactory(0x0101057a)="androidx.core.app.CoreComponentFactory" (Raw: "androidx.core.app.CoreComponentFactory")
                 E: activity (line=22)
                   A: android:theme(0x01010000)=@0x7f0a0000
                   A: android:name(0x01010003)="com.nanisamireddy.christain_lyrics.MainActivity" (Raw: "com.nanisamireddy.christain_lyrics.MainActivity")
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type 0x11)0x40003fb4
                   A: android:windowSoftInputMode(0x0101022b)=(type 0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type 0x12)0xffffffff
                   E: meta-data (line=36)
                     A: android:name(0x01010003)="io.flutter.embedding.android.NormalTheme" (Raw: "io.flutter.embedding.android.NormalTheme")
                     A: android:resource(0x01010025)=@0x7f0a0001
                   E: meta-data (line=46)
                     A: android:name(0x01010003)="io.flutter.embedding.android.SplashScreenDrawable" (Raw: "io.flutter.embedding.android.SplashScreenDrawable")
                     A: android:resource(0x01010025)=@0x7f040000
                   E: intent-filter (line=50)
                     E: action (line=51)
                       A: android:name(0x01010003)="android.intent.action.MAIN" (Raw: "android.intent.action.MAIN")
                     E: category (line=53)
                       A: android:name(0x01010003)="android.intent.category.LAUNCHER" (Raw: "android.intent.category.LAUNCHER")
                 E: meta-data (line=60)
                   A: android:name(0x01010003)="flutterEmbedding" (Raw: "flutterEmbedding")
                   A: android:value(0x01010024)=(type 0x10)0x2
[   +4 ms] Stopping app 'app.apk' on vivo 1917.
[   +2 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell am force-stop com.nanisamireddy.christain_lyrics
[ +179 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell pm list packages com.nanisamireddy.christain_lyrics
[ +127 ms] package:com.nanisamireddy.christain_lyrics
[   +4 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell cat /data/local/tmp/sky.com.nanisamireddy.christain_lyrics.sha1
[  +86 ms] fad10068617b7a811cbaa157c6480a24b2128457
[   +2 ms] Latest build already installed.
[   +1 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell -x logcat -v time -t 1
[ +181 ms] --------- beginning of main
                    04-26 17:02:03.545 W/babw    ( 4069): Empty content. Thus might mean that the settings are not synced down.
[  +25 ms] executing: C:\Users\NANI\AppData\Local\Android\sdk\platform-tools\adb.exe -s d053be10 shell am start -a android.intent.action.RUN -f 0x20000000 --ez enable-background-compilation true --ez
enable-dart-profiling true --ez enable-checked-mode true --ez verify-entry-points true com.nanisamireddy.christain_lyrics/com.nanisamireddy.christain_lyrics.MainActivity
[ +140 ms] Starting: Intent { act=android.intent.action.RUN flg=0x20000000 cmp=com.nanisamireddy.christain_lyrics/.MainActivity (has extras) }
[   +1 ms] Waiting for observatory port to be available...
[ +841 ms] I/flutter (12169): Observatory listening on ************************************
```
</details>

and i have a black screen on device....
my device vivo z1x with android 11 os
and my system windows 10...

**Comment by pedromassangocode** [comment]
Hi @nani-samireddy 
Is that an issue? Does the project get launched on the device after a while?

Please elaborate. It is not clear what the problem is!

**Comment by nani-samireddy** [comment]
yes....
it is coming while we are launching in the app on the device

**Comment by nani-samireddy** [comment]
it able to build the apk after the while the installation of the app in the device it is saying as waiting for the observatory port to be available  and then observatory listening ************
that stays forever.....

**Comment by sageata** [comment]
For me it happens on iOS since I installed latest version of xCode for iOS 14.5.

**Comment by iamshaunwu** [comment]
@nani-samireddy curious, which mobile brand and model are you using? I am having this issue on vivo X60 pro+.

**Comment by iamshaunwu** [comment]
I communicated with Vivo and they mentioned that they were able to install and run on debug without encountering the black screen problem. They highlighted that it has something to do with the "adb" and not vivo phone... I am stuck. ZZZZ...

**Comment by nani-samireddy** [comment]
Did you get the solution or not !!!...
And one-thing normal apps with java are running fine.. only with flutter apps..

**Comment by iamshaunwu** [comment]
> Did you get the solution or not !!!...
> And one-thing normal apps with java are running fine.. only with flutter apps..

No. Vivo said the issue probably lies with my ADB environment which they cannot resolve. **I don't have a solution.** I will probably sell away this vivo phone and try on Oppo (which is working fine) or apple phone.

**Comment by sulthanalihsan** [comment]
any solution for this issue?

**Comment by nani-samireddy** [comment]
still waiting...........

**Comment by MujmillahammedDafedar** [comment]
I am also same issue facing after update vivo v20 device. Is there any solution for this @pedromassangocode  @nani-samireddy

**Comment by nani-samireddy** [comment]
Flutter run --release is working

**Comment by nani-samireddy** [comment]
But that doesn't have hot reload

**Comment by Dev-Mahbub** [comment]
Is there any solution. It was not before. I am using vivo v20
√ Built build\app\outputs\flutter-apk\app-debug.apk.
I/flutter (13189): Observatory listening on ************************************

**Comment by nani-samireddy** [comment]
After the android 11 update it show this update...

**Comment by MujmillahammedDafedar** [comment]
> Is there any solution. It was not before. I am using vivo v20
> √ Built build\app\outputs\flutter-apk\app-debug.apk.
> I/flutter (13189): Observatory listening on ************************************

did you got any solution? I am also using vivo20 but facing same problems.

**Comment by lifeShadow59** [comment]
device brand/model - iqoo 7 legend
os version - android 11 ( Funtouch OD 11.1 Global )

black screen after installing APK.
the screen appears only after you stop debug mode and reopen the app.


Built build\app\outputs\flutter-apk\app-debug.apk.
I/flutter ( 3020): Observatory listening on ************************************

**Comment by nani-samireddy** [comment]
> > Is there any solution. It was not before. I am using vivo v20
> > √ Built build\app\outputs\flutter-apk\app-debug.apk.
> > I/flutter (13189): Observatory listening on ************************************
> 
> did you get any solution? I am also using vivo20 but facing the same problems.

no

**Comment by TahaTesser** [comment]
Hi everyone
I see there's an open issue addressing the case you described.
Please follow up on that issue, Closing as duplicate of https://github.com/flutter/flutter/issues/82520
If you disagree, please write in the comments and I will reopen it.
Thank you
